tests/conftest.py

```diff
@@ -1,11 +1,16 @@
+from typing import Any
+
 import pathlib
```

Comment:
```suggestion
from typing import Any
import pathlib
```
We have so far used PEP8 group import (https://www.python.org/dev/peps/pep-0008/#imports).
1. Standard libraries.
2. External libraries.
3. Same Package submodules.